### ISSUE TITLE
Metrics aren't being exported to Sumologic

### ISSUE THREAD

**Issue by natehitze** [SumoLogic/sumologic-otel-collector#690]
I'm new to OpenTelemetry and SumoLogic so I'm sure the issue is on my side I'm just having trouble pinning down what that issue is. I've verified that logs are sent to SumoLogic as they show up in a `_collector=...` query. 

I'm pretty certain otelcol-sumo is receiving metrics since `logging` is logging information about them to the console. I can't find any metrics in my SumoLogic account though. I'm using the same `_collector=...` query in a metrics search and it's empty. I've waited a couple hours to try to make sure it isn't just a delay in the data appearing. Do I have something configured incorrectly? Is there additional logging I can turn on to find more information? 

config.yaml
```yaml
exporters:
  sumologic:
    sending_queue:
      enabled: true
      persistent_storage_enabled: true
  logging:
    loglevel: debug

extensions:
  file_storage:
    directory: /etc/otelcol-sumo/storage
  sumologic:
    install_token: $TOKEN
    collector_name: $COLLECTORNAME

receivers:
  otlp:
    protocols:
      grpc:
        tls:
          cert_file: $CERTFILE
          key_file: $CERTKEY

service:
  extensions: [sumologic, file_storage]
  pipelines:
    logs:
      receivers: [otlp]
      exporters: [sumologic]
    metrics:
      receivers: [otlp]
      exporters: [sumologic, logging]
    traces:
      receivers: [otlp]
      exporters: [sumologic]
```

otelcol-sumo console logs for metrics
```
2022-08-02T21:06:11.996Z        info    service/collector.go:215        Starting otelcol-sumo...        {"Version": "v0.56.0-sumo-0", "NumCPU": 2}
2022-08-02T21:06:11.996Z        info    service/collector.go:128        Everything is ready. Begin running and processing data.
2022-08-02T21:06:33.659Z        info    MetricsExporter {"kind": "exporter", "data_type": "metrics", "name": "logging", "#metrics": 1}
2022-08-02T21:06:33.659Z        info    ResourceMetrics #0
Resource SchemaURL:
Resource labels:
     -> service.name: STRING(MyProduct)
     -> environment: STRING(development)
     -> telemetry.sdk.name: STRING(opentelemetry)
     -> telemetry.sdk.language: STRING(dotnet)
     -> telemetry.sdk.version: STRING(1.3.0.519)
ScopeMetrics #0
ScopeMetrics SchemaURL:
InstrumentationScope OpenTelemetry.Instrumentation.AspNetCore 1.0.0.0
Metric #0
Descriptor:
     -> Name: http.server.duration
     -> Description: measures the duration of the inbound HTTP request
     -> Unit: ms
     -> DataType: Histogram
     -> AggregationTemporality: AGGREGATION_TEMPORALITY_CUMULATIVE
HistogramDataPoints #0
Data point attributes:
     -> http.flavor: STRING(HTTP/2)
     -> http.host: STRING(localhost:5001)
     -> http.method: STRING(GET)
     -> http.scheme: STRING(https)
     -> http.status_code: STRING(200)
StartTimestamp: 2022-08-02 21:04:33.4550738 +0000 UTC
Timestamp: 2022-08-02 21:06:33.4374684 +0000 UTC
Count: 4
Sum: 2796.478800
ExplicitBounds #0: 0.000000
ExplicitBounds #1: 5.000000
ExplicitBounds #2: 10.000000
ExplicitBounds #3: 25.000000
ExplicitBounds #4: 50.000000
ExplicitBounds #5: 75.000000
ExplicitBounds #6: 100.000000
ExplicitBounds #7: 250.000000
ExplicitBounds #8: 500.000000
ExplicitBounds #9: 1000.000000
Buckets #0, Count: 0
Buckets #1, Count: 0
Buckets #2, Count: 0
Buckets #3, Count: 0
Buckets #4, Count: 1
Buckets #5, Count: 1
Buckets #6, Count: 0
Buckets #7, Count: 1
Buckets #8, Count: 0
Buckets #9, Count: 0
Buckets #10, Count: 1
        {"kind": "exporter", "data_type": "metrics", "name": "logging"}
2022-08-02T21:07:33.536Z        info    MetricsExporter {"kind": "exporter", "data_type": "metrics", "name": "logging", "#metrics": 1}
2022-08-02T21:07:33.537Z        info    ResourceMetrics #0
Resource SchemaURL:
Resource labels:
     -> service.name: STRING(MyProduct)
     -> environment: STRING(development)
     -> telemetry.sdk.name: STRING(opentelemetry)
     -> telemetry.sdk.language: STRING(dotnet)
     -> telemetry.sdk.version: STRING(1.3.0.519)
ScopeMetrics #0
ScopeMetrics SchemaURL:
InstrumentationScope OpenTelemetry.Instrumentation.AspNetCore 1.0.0.0
Metric #0
Descriptor:
     -> Name: http.server.duration
     -> Description: measures the duration of the inbound HTTP request
     -> Unit: ms
     -> DataType: Histogram
     -> AggregationTemporality: AGGREGATION_TEMPORALITY_CUMULATIVE
HistogramDataPoints #0
Data point attributes:
     -> http.flavor: STRING(HTTP/2)
     -> http.host: STRING(localhost:5001)
     -> http.method: STRING(GET)
     -> http.scheme: STRING(https)
     -> http.status_code: STRING(200)
StartTimestamp: 2022-08-02 21:04:33.4550738 +0000 UTC
Timestamp: 2022-08-02 21:07:33.4415283 +0000 UTC
Count: 4
Sum: 2796.478800
ExplicitBounds #0: 0.000000
ExplicitBounds #1: 5.000000
ExplicitBounds #2: 10.000000
ExplicitBounds #3: 25.000000
ExplicitBounds #4: 50.000000
ExplicitBounds #5: 75.000000
ExplicitBounds #6: 100.000000
ExplicitBounds #7: 250.000000
ExplicitBounds #8: 500.000000
ExplicitBounds #9: 1000.000000
Buckets #0, Count: 0
Buckets #1, Count: 0
Buckets #2, Count: 0
Buckets #3, Count: 0
Buckets #4, Count: 1
Buckets #5, Count: 1
Buckets #6, Count: 0
Buckets #7, Count: 1
Buckets #8, Count: 0
Buckets #9, Count: 0
Buckets #10, Count: 1
        {"kind": "exporter", "data_type": "metrics", "name": "logging"}
2022-08-02T21:08:33.534Z        info    MetricsExporter {"kind": "exporter", "data_type": "metrics", "name": "logging", "#metrics": 1}
2022-08-02T21:08:33.534Z        info    ResourceMetrics #0
Resource SchemaURL:
Resource labels:
     -> service.name: STRING(MyProduct)
     -> environment: STRING(development)
     -> telemetry.sdk.name: STRING(opentelemetry)
     -> telemetry.sdk.language: STRING(dotnet)
     -> telemetry.sdk.version: STRING(1.3.0.519)
ScopeMetrics #0
ScopeMetrics SchemaURL:
InstrumentationScope OpenTelemetry.Instrumentation.AspNetCore 1.0.0.0
Metric #0
Descriptor:
     -> Name: http.server.duration
     -> Description: measures the duration of the inbound HTTP request
     -> Unit: ms
     -> DataType: Histogram
     -> AggregationTemporality: AGGREGATION_TEMPORALITY_CUMULATIVE
HistogramDataPoints #0
Data point attributes:
     -> http.flavor: STRING(HTTP/2)
     -> http.host: STRING(localhost:5001)
     -> http.method: STRING(GET)
     -> http.scheme: STRING(https)
     -> http.status_code: STRING(200)
StartTimestamp: 2022-08-02 21:04:33.4550738 +0000 UTC
Timestamp: 2022-08-02 21:08:33.4385703 +0000 UTC
Count: 4
Sum: 2796.478800
ExplicitBounds #0: 0.000000
ExplicitBounds #1: 5.000000
ExplicitBounds #2: 10.000000
ExplicitBounds #3: 25.000000
ExplicitBounds #4: 50.000000
ExplicitBounds #5: 75.000000
ExplicitBounds #6: 100.000000
ExplicitBounds #7: 250.000000
ExplicitBounds #8: 500.000000
ExplicitBounds #9: 1000.000000
Buckets #0, Count: 0
Buckets #1, Count: 0
Buckets #2, Count: 0
Buckets #3, Count: 0
Buckets #4, Count: 1
Buckets #5, Count: 1
Buckets #6, Count: 0
Buckets #7, Count: 1
Buckets #8, Count: 0
Buckets #9, Count: 0
Buckets #10, Count: 1
        {"kind": "exporter", "data_type": "metrics", "name": "logging"}
2022-08-02T21:09:33.548Z        info    MetricsExporter {"kind": "exporter", "data_type": "metrics", "name": "logging", "#metrics": 1}
2022-08-02T21:09:33.548Z        info    ResourceMetrics #0
Resource SchemaURL:
Resource labels:
     -> service.name: STRING(MyProduct)
     -> environment: STRING(development)
     -> telemetry.sdk.name: STRING(opentelemetry)
     -> telemetry.sdk.language: STRING(dotnet)
     -> telemetry.sdk.version: STRING(1.3.0.519)
ScopeMetrics #0
ScopeMetrics SchemaURL:
InstrumentationScope OpenTelemetry.Instrumentation.AspNetCore 1.0.0.0
Metric #0
Descriptor:
     -> Name: http.server.duration
     -> Description: measures the duration of the inbound HTTP request
     -> Unit: ms
     -> DataType: Histogram
     -> AggregationTemporality: AGGREGATION_TEMPORALITY_CUMULATIVE
HistogramDataPoints #0
Data point attributes:
     -> http.flavor: STRING(HTTP/2)
     -> http.host: STRING(localhost:5001)
     -> http.method: STRING(GET)
     -> http.scheme: STRING(https)
     -> http.status_code: STRING(200)
StartTimestamp: 2022-08-02 21:04:33.4550738 +0000 UTC
Timestamp: 2022-08-02 21:09:33.4513333 +0000 UTC
Count: 4
Sum: 2796.478800
ExplicitBounds #0: 0.000000
ExplicitBounds #1: 5.000000
ExplicitBounds #2: 10.000000
ExplicitBounds #3: 25.000000
ExplicitBounds #4: 50.000000
ExplicitBounds #5: 75.000000
ExplicitBounds #6: 100.000000
ExplicitBounds #7: 250.000000
ExplicitBounds #8: 500.000000
ExplicitBounds #9: 1000.000000
Buckets #0, Count: 0
Buckets #1, Count: 0
Buckets #2, Count: 0
Buckets #3, Count: 0
Buckets #4, Count: 1
Buckets #5, Count: 1
Buckets #6, Count: 0
Buckets #7, Count: 1
Buckets #8, Count: 0
Buckets #9, Count: 0
Buckets #10, Count: 1
        {"kind": "exporter", "data_type": "metrics", "name": "logging"}
2022-08-02T21:10:33.544Z        info    MetricsExporter {"kind": "exporter", "data_type": "metrics", "name": "logging", "#metrics": 1}
2022-08-02T21:10:33.545Z        info    ResourceMetrics #0
Resource SchemaURL:
Resource labels:
     -> service.name: STRING(MyProduct)
     -> environment: STRING(development)
     -> telemetry.sdk.name: STRING(opentelemetry)
     -> telemetry.sdk.language: STRING(dotnet)
     -> telemetry.sdk.version: STRING(1.3.0.519)
ScopeMetrics #0
ScopeMetrics SchemaURL:
InstrumentationScope OpenTelemetry.Instrumentation.AspNetCore 1.0.0.0
Metric #0
Descriptor:
     -> Name: http.server.duration
     -> Description: measures the duration of the inbound HTTP request
     -> Unit: ms
     -> DataType: Histogram
     -> AggregationTemporality: AGGREGATION_TEMPORALITY_CUMULATIVE
HistogramDataPoints #0
Data point attributes:
     -> http.flavor: STRING(HTTP/2)
     -> http.host: STRING(localhost:5001)
     -> http.method: STRING(GET)
     -> http.scheme: STRING(https)
     -> http.status_code: STRING(200)
StartTimestamp: 2022-08-02 21:04:33.4550738 +0000 UTC
Timestamp: 2022-08-02 21:10:33.4484113 +0000 UTC
Count: 4
Sum: 2796.478800
ExplicitBounds #0: 0.000000
ExplicitBounds #1: 5.000000
ExplicitBounds #2: 10.000000
ExplicitBounds #3: 25.000000
ExplicitBounds #4: 50.000000
ExplicitBounds #5: 75.000000
ExplicitBounds #6: 100.000000
ExplicitBounds #7: 250.000000
ExplicitBounds #8: 500.000000
ExplicitBounds #9: 1000.000000
Buckets #0, Count: 0
Buckets #1, Count: 0
Buckets #2, Count: 0
Buckets #3, Count: 0
Buckets #4, Count: 1
Buckets #5, Count: 1
Buckets #6, Count: 0
Buckets #7, Count: 1
Buckets #8, Count: 0
Buckets #9, Count: 0
Buckets #10, Count: 1
        {"kind": "exporter", "data_type": "metrics", "name": "logging"}
2022-08-02T21:11:33.535Z        info    MetricsExporter {"kind": "exporter", "data_type": "metrics", "name": "logging", "#metrics": 1}
2022-08-02T21:11:33.535Z        info    ResourceMetrics #0
Resource SchemaURL:
Resource labels:
     -> service.name: STRING(MyProduct)
     -> environment: STRING(development)
     -> telemetry.sdk.name: STRING(opentelemetry)
     -> telemetry.sdk.language: STRING(dotnet)
     -> telemetry.sdk.version: STRING(1.3.0.519)
ScopeMetrics #0
ScopeMetrics SchemaURL:
InstrumentationScope OpenTelemetry.Instrumentation.AspNetCore 1.0.0.0
Metric #0
Descriptor:
     -> Name: http.server.duration
     -> Description: measures the duration of the inbound HTTP request
     -> Unit: ms
     -> DataType: Histogram
     -> AggregationTemporality: AGGREGATION_TEMPORALITY_CUMULATIVE
HistogramDataPoints #0
Data point attributes:
     -> http.flavor: STRING(HTTP/2)
     -> http.host: STRING(localhost:5001)
     -> http.method: STRING(GET)
     -> http.scheme: STRING(https)
     -> http.status_code: STRING(200)
StartTimestamp: 2022-08-02 21:04:33.4550738 +0000 UTC
Timestamp: 2022-08-02 21:11:33.4402284 +0000 UTC
Count: 4
Sum: 2796.478800
ExplicitBounds #0: 0.000000
ExplicitBounds #1: 5.000000
ExplicitBounds #2: 10.000000
ExplicitBounds #3: 25.000000
ExplicitBounds #4: 50.000000
ExplicitBounds #5: 75.000000
ExplicitBounds #6: 100.000000
ExplicitBounds #7: 250.000000
ExplicitBounds #8: 500.000000
ExplicitBounds #9: 1000.000000
Buckets #0, Count: 0
Buckets #1, Count: 0
Buckets #2, Count: 0
Buckets #3, Count: 0
Buckets #4, Count: 1
Buckets #5, Count: 1
Buckets #6, Count: 0
Buckets #7, Count: 1
Buckets #8, Count: 0
Buckets #9, Count: 0
Buckets #10, Count: 1
        {"kind": "exporter", "data_type": "metrics", "name": "logging"}
2022-08-02T21:12:33.534Z        info    MetricsExporter {"kind": "exporter", "data_type": "metrics", "name": "logging", "#metrics": 1}
2022-08-02T21:12:33.534Z        info    ResourceMetrics #0
Resource SchemaURL:
Resource labels:
     -> service.name: STRING(MyProduct)
     -> environment: STRING(development)
     -> telemetry.sdk.name: STRING(opentelemetry)
     -> telemetry.sdk.language: STRING(dotnet)
     -> telemetry.sdk.version: STRING(1.3.0.519)
ScopeMetrics #0
ScopeMetrics SchemaURL:
InstrumentationScope OpenTelemetry.Instrumentation.AspNetCore 1.0.0.0
Metric #0
Descriptor:
     -> Name: http.server.duration
     -> Description: measures the duration of the inbound HTTP request
     -> Unit: ms
     -> DataType: Histogram
     -> AggregationTemporality: AGGREGATION_TEMPORALITY_CUMULATIVE
HistogramDataPoints #0
Data point attributes:
     -> http.flavor: STRING(HTTP/2)
     -> http.host: STRING(localhost:5001)
     -> http.method: STRING(GET)
     -> http.scheme: STRING(https)
     -> http.status_code: STRING(200)
StartTimestamp: 2022-08-02 21:04:33.4550738 +0000 UTC
Timestamp: 2022-08-02 21:12:33.4377587 +0000 UTC
Count: 4
Sum: 2796.478800
ExplicitBounds #0: 0.000000
ExplicitBounds #1: 5.000000
ExplicitBounds #2: 10.000000
ExplicitBounds #3: 25.000000
ExplicitBounds #4: 50.000000
ExplicitBounds #5: 75.000000
ExplicitBounds #6: 100.000000
ExplicitBounds #7: 250.000000
ExplicitBounds #8: 500.000000
ExplicitBounds #9: 1000.000000
Buckets #0, Count: 0
Buckets #1, Count: 0
Buckets #2, Count: 0
Buckets #3, Count: 0
Buckets #4, Count: 1
Buckets #5, Count: 1
Buckets #6, Count: 0
Buckets #7, Count: 1
Buckets #8, Count: 0
Buckets #9, Count: 0
Buckets #10, Count: 1
        {"kind": "exporter", "data_type": "metrics", "name": "logging"}
^C2022-08-02T21:13:23.155Z      info    service/collector.go:159        Received signal from OS {"signal": "interrupt"}
2022-08-02T21:13:23.155Z        info    service/collector.go:231        Starting shutdown...
2022-08-02T21:13:23.155Z        info    pipelines/pipelines.go:118      Stopping receivers...
2022-08-02T21:13:23.187Z        info    pipelines/pipelines.go:125      Stopping processors...
2022-08-02T21:13:23.188Z        info    pipelines/pipelines.go:132      Stopping exporters...
2022-08-02T21:13:23.188Z        info    extensions/extensions.go:56     Stopping extensions...
```

**Comment by swiatekm-sumo** [comment]
Your configuration looks correct to me. Can you post the full logs from otel startup? There should be information about what the extension is doing there.

Relatedly, can you see the collector in the Collectors list in your Sumo account?

**Comment by natehitze** [comment]
Sure thing, here it is. Yes the collector is in my Sumo account. Logs forward correctly from my application -> otelcol-sumo (this collector) -> Sumo. I'll include screenshots from Sumo below as well.

```2022-08-05T13:20:46.847Z        info    service/telemetry.go:102        Setting up own telemetry...
2022-08-05T13:20:46.848Z        info    service/telemetry.go:137        Serving Prometheus metrics      {"address": ":8888", "level": "basic"}
2022-08-05T13:20:46.848Z        info    components/components.go:30     Stability level of component is undefined      {"kind": "exporter", "data_type": "logs", "name": "sumologic", "stability": "undefined"}
2022-08-05T13:20:46.848Z        warn    sumologicexporter@v0.56.0-sumo-0/exporter.go:81
***********************************************************************************************************************************************************
***    Translating attributes is deprecated and is going to be dropped soon. Please see the migration document:
                                    ***
***    https://github.com/SumoLogic/sumologic-otel-collector/blob/main/docs/Upgrading.md#sumologic-exporter-drop-support-for-translating-attributes.    ***
***********************************************************************************************************************************************************
        {"kind": "exporter", "data_type": "logs", "name": "sumologic"}
2022-08-05T13:20:46.851Z        warn    sumologicexporter@v0.56.0-sumo-0/exporter.go:89
***********************************************************************************************************************************************************
***    Translating Telegraf metric names is deprecated and is going to be dropped soon. Please see the migration document:                              ***
***    https://github.com/SumoLogic/sumologic-otel-collector/blob/main/docs/Upgrading.md#sumologic-exporter-drop-support-for-translating-attributes.    ***
***********************************************************************************************************************************************************
        {"kind": "exporter", "data_type": "logs", "name": "sumologic"}
2022-08-05T13:20:46.851Z        info    sumologicexporter@v0.56.0-sumo-0/exporter.go:119        Sumo Logic Exporter configured  {"kind": "exporter", "data_type": "logs", "name": "sumologic", "log_format": "otlp", "metric_format": "otlp", "trace_format": "otlp"}
2022-08-05T13:20:46.851Z        info    components/components.go:30     Stability level of component is undefined      {"kind": "exporter", "data_type": "metrics", "name": "sumologic", "stability": "undefined"}
2022-08-05T13:20:46.851Z        warn    sumologicexporter@v0.56.0-sumo-0/exporter.go:81
***********************************************************************************************************************************************************
***    Translating attributes is deprecated and is going to be dropped soon. Please see the migration document:
                                    ***
***    https://github.com/SumoLogic/sumologic-otel-collector/blob/main/docs/Upgrading.md#sumologic-exporter-drop-support-for-translating-attributes.    ***
***********************************************************************************************************************************************************
        {"kind": "exporter", "data_type": "metrics", "name": "sumologic"}
2022-08-05T13:20:46.851Z        warn    sumologicexporter@v0.56.0-sumo-0/exporter.go:89
***********************************************************************************************************************************************************
***    Translating Telegraf metric names is deprecated and is going to be dropped soon. Please see the migration document:                              ***
***    https://github.com/SumoLogic/sumologic-otel-collector/blob/main/docs/Upgrading.md#sumologic-exporter-drop-support-for-translating-attributes.    ***
***********************************************************************************************************************************************************
        {"kind": "exporter", "data_type": "metrics", "name": "sumologic"}
2022-08-05T13:20:46.851Z        info    sumologicexporter@v0.56.0-sumo-0/exporter.go:119        Sumo Logic Exporter configured  {"kind": "exporter", "data_type": "metrics", "name": "sumologic", "log_format": "otlp", "metric_format": "otlp", "trace_format": "otlp"}
2022-08-05T13:20:46.851Z        info    components/components.go:30     In development component. May change in the future.     {"kind": "exporter", "data_type": "metrics", "name": "logging", "stability": "in development"}
2022-08-05T13:20:46.852Z        info    components/components.go:30     Stability level of component is undefined      {"kind": "exporter", "data_type": "traces", "name": "sumologic", "stability": "undefined"}
2022-08-05T13:20:46.852Z        warn    sumologicexporter@v0.56.0-sumo-0/exporter.go:81
***********************************************************************************************************************************************************
***    Translating attributes is deprecated and is going to be dropped soon. Please see the migration document:
                                    ***
***    https://github.com/SumoLogic/sumologic-otel-collector/blob/main/docs/Upgrading.md#sumologic-exporter-drop-support-for-translating-attributes.    ***
***********************************************************************************************************************************************************
        {"kind": "exporter", "data_type": "traces", "name": "sumologic"}
2022-08-05T13:20:46.852Z        warn    sumologicexporter@v0.56.0-sumo-0/exporter.go:89
***********************************************************************************************************************************************************
***    Translating Telegraf metric names is deprecated and is going to be dropped soon. Please see the migration document:                              ***
***    https://github.com/SumoLogic/sumologic-otel-collector/blob/main/docs/Upgrading.md#sumologic-exporter-drop-support-for-translating-attributes.    ***
***********************************************************************************************************************************************************
        {"kind": "exporter", "data_type": "traces", "name": "sumologic"}
2022-08-05T13:20:46.852Z        info    sumologicexporter@v0.56.0-sumo-0/exporter.go:119        Sumo Logic Exporter configured  {"kind": "exporter", "data_type": "traces", "name": "sumologic", "log_format": "otlp", "metric_format": "otlp", "trace_format": "otlp"}
2022-08-05T13:20:46.852Z        info    extensions/extensions.go:42     Starting extensions...
2022-08-05T13:20:46.852Z        info    extensions/extensions.go:45     Extension is starting...        {"kind": "extension", "name": "sumologic"}
2022-08-05T13:20:46.853Z        info    credentials/credentialsstore_localfs.go:146     Collector registration credentials retrieved from local fs      {"kind": "extension", "name": "sumologic", "path": "/root/.sumologic-otel-collector/7d22c4b43d0336303452c0d9ecd356b1dbfb149651c3117a436a20de7ea4fbbb"}
2022-08-05T13:20:46.856Z        info    sumologicextension@v0.56.0-sumo-0/extension.go:188      Validating collector credentials...     {"kind": "extension", "name": "sumologic", "collector_credential_id": "AnTNXeXKT2PecxZIkAaU", "collector_id": "0000000010C3388E"}
2022-08-05T13:20:47.430Z        info    sumologicextension@v0.56.0-sumo-0/extension.go:255      Found stored credentials, skipping registration {"kind": "extension", "name": "sumologic", "collector_name": "myproduct"}
2022-08-05T13:20:47.430Z        info    extensions/extensions.go:49     Extension started.      {"kind": "extension", "name": "sumologic"}
2022-08-05T13:20:47.430Z        info    extensions/extensions.go:45     Extension is starting...        {"kind": "extension", "name": "file_storage"}
2022-08-05T13:20:47.430Z        info    extensions/extensions.go:49     Extension started.      {"kind": "extension", "name": "file_storage"}
2022-08-05T13:20:47.430Z        info    pipelines/pipelines.go:74       Starting exporters...
2022-08-05T13:20:47.430Z        info    pipelines/pipelines.go:78       Exporter is starting... {"kind": "exporter", "data_type": "logs", "name": "sumologic"}
2022-08-05T13:20:47.430Z        info    sumologicextension@v0.56.0-sumo-0/extension.go:490      Heartbeat loop initialized. Starting to send hearbeat requests  {"kind": "extension", "name": "sumologic", "collector_name": "myproduct", "collector_id": "0000000010C3388E"}
2022-08-05T13:20:47.438Z        info    pipelines/pipelines.go:82       Exporter started.       {"kind": "exporter", "data_type": "logs", "name": "sumologic"}
2022-08-05T13:20:47.438Z        info    pipelines/pipelines.go:78       Exporter is starting... {"kind": "exporter", "data_type": "metrics", "name": "sumologic"}
2022-08-05T13:20:47.438Z        info    pipelines/pipelines.go:82       Exporter started.       {"kind": "exporter", "data_type": "metrics", "name": "sumologic"}
2022-08-05T13:20:47.438Z        info    pipelines/pipelines.go:78       Exporter is starting... {"kind": "exporter", "data_type": "metrics", "name": "logging"}
2022-08-05T13:20:47.438Z        info    pipelines/pipelines.go:82       Exporter started.       {"kind": "exporter", "data_type": "metrics", "name": "logging"}
2022-08-05T13:20:47.438Z        info    pipelines/pipelines.go:78       Exporter is starting... {"kind": "exporter", "data_type": "traces", "name": "sumologic"}
2022-08-05T13:20:47.438Z        info    pipelines/pipelines.go:82       Exporter started.       {"kind": "exporter", "data_type": "traces", "name": "sumologic"}
2022-08-05T13:20:47.438Z        info    pipelines/pipelines.go:86       Starting processors...
2022-08-05T13:20:47.438Z        info    pipelines/pipelines.go:98       Starting receivers...
2022-08-05T13:20:47.438Z        info    pipelines/pipelines.go:102      Receiver is starting... {"kind": "receiver", "name": "otlp", "pipeline": "logs"}
2022-08-05T13:20:47.441Z        info    otlpreceiver/otlp.go:70 Starting GRPC server on endpoint 0.0.0.0:4317   {"kind": "receiver", "name": "otlp", "pipeline": "logs"}
2022-08-05T13:20:47.453Z        info    pipelines/pipelines.go:106      Receiver started.       {"kind": "receiver", "name": "otlp", "pipeline": "logs"}
2022-08-05T13:20:47.453Z        info    pipelines/pipelines.go:102      Receiver is starting... {"kind": "receiver", "name": "otlp", "pipeline": "metrics"}
2022-08-05T13:20:47.453Z        info    pipelines/pipelines.go:106      Receiver started.       {"kind": "receiver", "name": "otlp", "pipeline": "metrics"}
2022-08-05T13:20:47.453Z        info    pipelines/pipelines.go:102      Receiver is starting... {"kind": "receiver", "name": "otlp", "pipeline": "traces"}
2022-08-05T13:20:47.453Z        info    pipelines/pipelines.go:106      Receiver started.       {"kind": "receiver", "name": "otlp", "pipeline": "traces"}
2022-08-05T13:20:47.453Z        info    service/collector.go:215        Starting otelcol-sumo...        {"Version": "v0.56.0-sumo-0", "NumCPU": 2}
2022-08-05T13:20:47.453Z        info    service/collector.go:128        Everything is ready. Begin running and processing data.
2022-08-05T13:20:53.542Z      info    service/collector.go:159        Received signal from OS {"signal": "interrupt"}
2022-08-05T13:20:53.542Z        info    service/collector.go:231        Starting shutdown...
2022-08-05T13:20:53.542Z        info    pipelines/pipelines.go:118      Stopping receivers...
2022-08-05T13:20:53.542Z        info    pipelines/pipelines.go:125      Stopping processors...
2022-08-05T13:20:53.542Z        info    pipelines/pipelines.go:132      Stopping exporters...
2022-08-05T13:20:53.542Z        info    extensions/extensions.go:56     Stopping extensions...
2022-08-05T13:20:53.542Z        info    sumologicextension@v0.56.0-sumo-0/extension.go:495      Heartbeat sender turned off     {"kind": "extension", "name": "sumologic", "collector_name": "myproduct", "collector_id": "0000000010C3388E"}
```

![image](https://user-images.githubusercontent.com/3506334/183087986-7a39e873-2bef-48f8-98e8-4f2f15f36473.png)

![image](https://user-images.githubusercontent.com/3506334/183088303-38b5d640-9965-48b1-a3a4-a8d3bb5a4216.png)

**Comment by swiatekm-sumo** [comment]
Just to make sure we're not getting wires crossed here, your screenshot shows the collector name to end in `-otel` whereas in the log the name is `myproduct`. Is the latter value just an example?

**Comment by natehitze** [comment]
Yes, sorry! They are the same collector name for sure, I just masked them differently.

**Comment by swiatekm-sumo** [comment]
Is it possible that you're only pushing histograms? Sumo doesn't support Histograms via OTLP yet, and I can only see histograms in our log. Can you try with something simple like the `hostmetrics` receiver and see if data shows up then?

**Comment by natehitze** [comment]
Yes, that's definitely possible. I was just pushing whatever default automatic metric(s) that came out of the OTEL package I added to my project. I didn't see any notes about histograms not being supported so I never thought to check which metrics were being sent -- I was just trying to get something quick setup to test it. We ended up not choosing Sumo (or OTEL in general). Thanks for looking into this.

By the way, the install documentation was incredibly helpful. I think I spent more time getting TLS setup than installing and configuring the collector.
